domain/src/main/java/org/oppia/domain/oppialogger/OppiaLogger.kt

```diff
@@ -5,14 +5,14 @@ import org.oppia.app.model.EventLog.EventAction
 import org.oppia.domain.oppialogger.analytics.AnalyticsController
 import javax.inject.Inject
 
-/** Logger that handles exceptions, crashes, events, and console logging. */
+/** Logger that handles event logging.
```

Comment:
Please fix the formatting here. Should be:

```kotlin
/**
 * Logger that handles event logging.
 *
 * All classes that want to log events should use this class rather than [AnalyticsController].
 */
```

Though, in terms of that last bit, maybe it's better placed in AnalyticsController?  I suspect that if someone is reading OppiaLogger's documentation, they may already know that they want to use this class whereas we want to warn people away from using AnalyticsController.

Reply:
That makes sense, I will add the last part in AnalyticsController

Reply:
Resolving this!